code/XMLTester.xml

```diff
@@ -2915,6 +2915,20 @@
 </participant>
 <!--><-->
 
+<!-->PULL_MODE_DATAWRITER<-->
```

Comment:
XML snippet should be complete. Example: https://github.com/eProsima/Fast-DDS-docs/blob/master/code/XMLTester.xml#L447

Reply:
Done on 0ae2c18ad2fd4215f118b771c77076e5318b62f1